pkgs/os-specific/linux/firmware/fwupd/default.nix

```diff
@@ -58,6 +58,7 @@ let
   python = python3.withPackages (p: with p; [
     pygobject3
     setuptools
+    markdown
```

Comment:
We should probably patch out the markdown check like we did previously.

Reply:
Oh, good point, I should have read the comments I deleted :woman_facepalming: